src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
```

Comment:
For now in App.js, you can return a simple heading, remove all this returned code as we won't be needing it